amount.go

```diff
@@ -112,7 +143,6 @@ func (a Amount) Convert(currencyCode, rate string) (Amount, error) {
 	}
 	ctx := apd.BaseContext.WithPrecision(16)
 	ctx.Mul(result, a.number, result)
-
```

Comment:
Let's not revert my code formatting choices. I like having a newline before the return.